### DISCUSSION TITLE
Display all XMP and EXPO profiles

### DISCUSSION THREAD

**Discussion by Jiematcha** [memtest86plus/memtest86plus#458] [Enhancement]
I have enabled EXPO DDR5 6000，but the test show XMP 6900.
AMD 7600
ASUS B650E-E
ACER 32G EXPO 6000

**Comment by debrouxl** [comment]
memtest86+ does not contain EXPO parsing code for SPD data yet; also, memtest86+ 6.00-6.20 typically reports the timings and frequency for one of the XMP settings found within the SPD data, which may or may not match the profile chosen by the user.
However, for your Zen 4 desktop processor, I'd say that the development version of memtest86+ ought to correctly report the active RAM settings, like it does on the Zen 3 desktop processor mentioned in #324 . See that issue for a link to the download page for a nightly build of the development version :)

**Comment by x86fr** [comment]
I have plan to display debug informations and all XMP/EXPO profiles in the menu. That would be a nice features.
